rasa/nlu/components.py

```diff
@@ -88,11 +88,31 @@ def validate_required_components_from_data(
 ):
 
     response_selector_exists = False
+    regex_config_exists = False
+    crfentity_config_exists = False
+
     for component in pipeline:
+        if "RegexFeaturizer" in component.name:
+            regex_config_exists = True
+        if "CRFEntityExtractor" in component.name:
+            crfentity_config_exists = True
         # check if a response selector is part of NLU pipeline
         if MESSAGE_RESPONSE_ATTRIBUTE in component.provides:
             response_selector_exists = True
 
+    # check if there are regex features
+    if data.regex_features and not regex_config_exists:
+        warnings.warn(
+            "Training data consists of regex examples but "
+            "no RegexFeaturizer setup in config.yml"
+        )
```

Comment:
maybe model the warnings after https://github.com/RasaHQ/rasa/blob/master/rasa/core/policies/form_policy.py#L50

Reply:
ok I'll look at that one I was just basing it off the original one that was already there about the response selector

Reply:
No prob, it's always a good idea to suggest to the user how to fix the warning

Reply:
@erohmensing would this be better or too much?

```
"You have defined training data with Regex's, but "
"haven't setup RegexFeaturizer in your config. "
"Either add RegexFeaturizer to your config or "
"remove the regex training data"
```